packages/app/src/visualizer/utils.ts

```diff
@@ -102,11 +102,12 @@ function getSupportedNxVis(
 
   const dataset = findSignalDataset(entity, attrValuesStore);
   const isCplx = hasComplexType(dataset);
-  const { interpretation } = attrValuesStore.get(dataset);
+  const { interpretation, CLASS } = attrValuesStore.get(dataset);
 
   if (
-    interpretation === NxInterpretation.RGB &&
-    hasNumDims(dataset, 3) &&
+    (interpretation === NxInterpretation.RGB || CLASS === 'IMAGE') &&
```

Comment:
Oh forgot to point out here that I'm adding a check for `CLASS === 'IMAGE'` on the signal. I reckon that's why Sandor thought we didn't support RGB images with NeXus: maybe they don't use `interpretation=rgb-image` at FairMAT.